Formula/postgresql@12.rb

```diff
@@ -81,22 +81,87 @@ def post_install
     return if ENV["CI"]
 
     (var/"log").mkpath
-    (var/"postgres").mkpath
-    unless File.exist? "#{var}/postgres/PG_VERSION"
-      system "#{bin}/initdb", "--locale=C", "-E", "UTF-8", "#{var}/postgres"
+    (var/name).mkpath
+    unless File.exist? "#{var}/#{name}/PG_VERSION"
+      system "#{bin}/initdb", "--locale=C", "-E", "UTF-8", "#{var}/#{name}"
     end
   end
 
+  # Previous versions of this formula used the same data dir as the regular
+  # postgresql formula. So we check whether the versioned data dir exists
+  # and has a PG_VERSION file, which should indicate that the versioned
+  # data dir is in use. Otherwise, returns the old data dir path.
+  def postgresql_datadir
```

Comment:
This should be used in `post_install`, I think?

Reply:
So I didn't use this method, but I added a new helper method that always resolves to the versioned data dir path, based on my understanding that `post_install` is only used when the formula is installed. This way, I hope it's clear what the intended data dir path is supposed to be when someone is reading the formula source.

Addressed in 03dfafe1a4